src/Utils/HtmlUtils.php

```diff
@@ -9,36 +9,90 @@ class HtmlUtils {
    * cases.
    */
    public static function makeImageElement(string $class, string $src, int $width, int $height, string $alt, string $id = ''): string {
-      $elementParts = [
-         '<img',
-         "class=\"{$class}\"",
-         "src=\"{$src}\"",
-         "width=\"{$width}\"",
-         "height=\"{$height}\"",
-         "alt=\"{$alt}\""
+      $elPartParams = [
+         'class' => $class,
+         'src' => $src,
+         'width' => $width,
+         'height' => $height,
+         'alt' => $alt,
+         'id' => $id
       ];
-      if ($id) {
-         $elementParts[] = "id=\"$id\"";
-      }
-      $elementParts[] = ">";
-      return implode(' ', $elementParts);
+      $elPartStr = self::generateElementPartStr($elPartParams);
+      return "<img {$elPartStr} />";
+   }
+
+   public static function makeDivElement(string $text, array $elPartParams = []): string {
+      $elPartStr = self::generateElementPartStr($elPartParams);
+      return "<div {$elPartStr}>{$text}</div>";
    }
 
-   public static function makeSpanElement(string $text, string $class): string {
-      return "<span class=\"{$class}\">{$text}</span>";
+   public static function makeSpanElement(string $text, array $elPartParams): string {
+      $elPartStr = self::generateElementPartStr($elPartParams);
+      return "<span {$elPartStr}>{$text}</span>";
    }
 
    public static function makeH1Element(string $text, string $class): string {
-      return "<h1 class=\"{$class}\">{$text}</h1>";
+      $elParts = ['class' => $class];
+      return self::makeHXElement(1, $text, $elParts);
+   }
+
+   public static function makeH3Element(string $text, string $class = null): string {
+      $elParts = $class ? ['class' => $class] : [];
+      return self::makeHXElement(3, $text, $elParts);
+   }
+
+   public static function makeHXElement(int $hx, string $text, array $elPartParams = []): string {
+      $elPartStr = self::generateElementPartStr($elPartParams);
+      $startTag = "<h{$hx} {$elPartStr}>";
+      $endTag = "</h{$hx}>";
+      return "{$startTag}{$text}{$endTag}";
+   }
+
+   // This is pretty specifically for the MiniArticle Tag List
+   public static function makeUnorderList(array $listValues, $addDataMeta = false) {
```

Comment:
This is now unused. We use `<spans>` for this stuff, lists were just a old habit.

I might keep the code it b/c it was fun to build.